pipelines/fbc-release/README.md

```diff
@@ -17,6 +17,10 @@ Tekton release pipeline to interact with FBC Pipeline
 | taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                    | Yes       | https://github.com/redhat-appstudio/release-service-catalog.git |
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                           | Yes       | main                                                            |
 
+### Changes in 1.5.0
+- add a new task reference to the task `publish-index-image` and adjusts its task parameters
```

Comment:
If I didn't know about this PR, I wouldn't understand that it's about switching from using `create-internal-request` task to a dedicated task that uses the internal-request script. Can you modify this to say that?